xml/System.Threading/ThreadPool.xml

```diff
@@ -1268,6 +1268,9 @@ If a thread pool implementation may have different types of work items, the coun
   
  Setting the thread pool size too large can cause performance problems. If too many threads are executing at the same time, the task switching overhead becomes a significant factor.  
   
+> [!NOTE]
+>  If either argument's value is higher than 32767, then 32767 will be used instead inside the function. The function will return true if it's successful with the updated value(s) and therefore the new maximum number of worker threads or I/O completion threads might be lower than the corresponding argument.
```

Comment:
Not sure if it should say `short.MaxValue` or `32767` is fine. The code uses `short.MaxValue`.

Reply:
I don't think it's necessary to mention the actual value, it's just an implementation detail. How about something like this:

```suggestion
>  The thread pool may have upper limits for the maximum thread counts (such as `short.MaxValue`, depending on the implementation). The argument values are capped to the upper limit, so even when the method returns `true`, the actual maximum thread counts may be lower than requested.
```